sb_vision/cv3d.py

```diff
@@ -0,0 +1,72 @@
+"""
+Python wrapping around cffi layer to interop to OpenCV 3D functions.
+"""
+
+from typing import Sequence, Tuple
+
+from sb_vision.native import _cv3d  # type: ignore
+
+from .coordinates import Cartesian, PixelCoordinate
+
+
+class Cv3dError(RuntimeError):
+    """A 3D related OpenCV error."""
+
+    pass
+
+
+def _ffi_flattened_float_array(values: Sequence[Sequence[float]]):
+    if not values:
+        raise ValueError("Refusing to create ffi matrix for empty 'values'.")
+
+    lengths = list({len(x) for x in values})
+    if len(lengths) != 1:
+        raise ValueError(
+            "'values' is not rectangular (got row lengths of {} and {})".format(
```

Comment:
very very minor: this doesn't produce an error message with an oxford comma :smile: